### PR TITLE
feat: file upload

### PR DESCRIPTION
**Pull request by Oluwaseyi3**
I fixed the file change and naming path in the Input Field as required in the task

### CODE REVIEW

**Comment by AvineshTripathi** [comment]
Hey @Oluwaseyi3 thanks for the PR can you look into why the build failed, let me know when you are ready for review

**Comment by shivam-sharma7** [comment]
@Oluwaseyi3 use `3.0.x` instead `1.x`

**Comment by shivam-sharma7** [comment]
@Oluwaseyi3 This is peer dependencies issue see-https://github.com/tailwarden/komiser/actions/runs/6157526105/job/16708429236?pr=937#step:4:12

**Comment by Oluwaseyi3** [comment]
> @Oluwaseyi3 This is peer dependencies issue see-https://github.com/tailwarden/komiser/actions/runs/6157526105/job/16708429236?pr=937#step:4:12

oh thanks so much

**Comment by AvineshTripathi** [comment]
Hey @Oluwaseyi3 I tried the changes here are few things
- firstly in any source either by file or env var or anthing the submit button is not working for me 
- while doing submit I see error at my console 
- sometimes error in above images comes

**Comment by Oluwaseyi3** [comment]
> Hey @Oluwaseyi3 I tried the changes here are few things
> 
> * firstly in any source either by file or env var or anthing the submit button is not working for me
> * while doing submit I see error at my console
> * sometimes error in above images comes
>   


The task given to me was based on fixed the file path (as described in the code) but I can try working on that 
Screenshot 2023-09-13 101057

**Comment by AvineshTripathi** [comment]
I tried it locally and the file name change is not happening, it is always showing the default path

**Comment by Oluwaseyi3** [comment]
> I tried it locally and the file name change is not happening, it is always showing the default path

That is quite strange because it works perfectly here. I’ll crosscheck if I pushed the right commit. I suspect that may be the issue. I do that tonight

**Comment by Oluwaseyi3** [comment]
> I tried it locally and the file name change is not happening, it is always showing the default path

Hi @AvineshTripathi 
Can you check this one out? I pushed the right update
Screenshot 2023-09-14 101436

File: dashboard/pages/onboarding/provider/aws.tsx [hunk 1]
```diff
@@ -70,10 +72,18 @@ export default function AWSCredentials() {
...
   const handleFileChange = (event: any) => {
     const file = event.target.files[0];
+
+    if (fileInputRef.current && fileInputRef.current.files) {
+      const selectedFile = fileInputRef.current.files[0];
+      setFileName(selectedFile.name);
+      console.log(selectedFile);
+    }
+
     // TODO: (onboarding-wizard) handle file change and naming. Set Input field to file.name and use temporary file path for the upload value
```
Comment (AvineshTripathi) [inline dashboard/pages/onboarding/provider/aws.tsx:82]: Are you sure this is the commit with your changes cause still I dont see change in filename holder

**Comment by Oluwaseyi3** [comment]
Sorry @AvineshTripathi pls confirm this again. I've fixed it

File: dashboard/components/onboarding-wizard/LabelledInput.tsx [hunk 2]
```diff
@@ -19,8 +20,11 @@ function LabelledInput({
   value,
   subLabel,
   placeholder,
+  onChange,
   disabled = false
 }: LabelledInputProps) {
+  console.log(value);
```
Comment (AvineshTripathi) [inline dashboard/components/onboarding-wizard/LabelledInput.tsx:26]: we can remove this!

File: dashboard/pages/onboarding/provider/kubernetes.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ export default function KubernetesCredentials() {
   const handleFileChange = (event: any) => {
     const file = event.target.files[0];
     // TODO: (onboarding-wizard) handle file change and naming. Set Input field to file.name and use temporary file path for the upload value
```
Comment (AvineshTripathi) [inline dashboard/pages/onboarding/provider/kubernetes.tsx:56]: can we remove this?

File: dashboard/pages/onboarding/choose-database.tsx [hunk 4]
```diff
@@ -147,10 +150,11 @@ export default function ChooseDatabase() {
                 alt="Komiser Logo"
                 width={135}
                 height={100}
-                className={`${database === allDBProviders.POSTGRES
-                  ? 'ml-16'
-                  : 'rotate-onboarding-arrow ml-[182px]'
-                  }`}
+                className={`${
+                  database === allDBProviders.POSTGRES
```
Comment (AvineshTripathi) [inline dashboard/pages/onboarding/choose-database.tsx:154]: can you explain this to me, why are we particular about POSTGRESS and not mysql

**Review by AvineshTripathi — COMMENTED** [review]
Please rebase! and @ShubhamPalriwala can you look into frontend more before we merge

File: dashboard/components/onboarding-wizard/LabelledInput.tsx (hunk 2 above)
Comment (Oluwaseyi3) [reply dashboard/components/onboarding-wizard/LabelledInput.tsx:26]: Sure will do that!
@AvineshTripathi

**Comment by AvineshTripathi** [comment]
Hey @Oluwaseyi3 you will have to resolve conflicts for your PR as there was a recent merge https://github.com/tailwarden/komiser/pull/983 which did some chages to the files

**Comment by Oluwaseyi3** [comment]
> Hey @Oluwaseyi3 you will have to resolve conflicts for your PR as there was a recent merge #983 which did some chages to the files

Okay

**Pull request closed without merging**